services/spar/src/Spar/Sem/AssIDStore/Cassandra.hs

```diff
@@ -0,0 +1,29 @@
+module Spar.Sem.AssIDStore.Cassandra where
+
+import Cassandra
+import Control.Monad.Except (runExceptT)
+import Imports hiding (MonadReader (..), Reader)
+import Polysemy
+import Polysemy.Error
+import Polysemy.Reader
+import SAML2.WebSSO (fromTime, getNow)
+import qualified Spar.Data as Data
+import Spar.Sem.AReqIDStore.Cassandra ()
+import Spar.Sem.AssIDStore
+import Wire.API.User.Saml (Opts, TTLError)
+
+assIDStoreToCassandra ::
+  forall m r a.
+  (MonadClient m, Members '[Embed m, Error TTLError, Embed IO, Reader Opts] r) =>
+  Sem (AssIDStore ': r) a ->
+  Sem r a
+assIDStoreToCassandra =
+  interpret $ \case
+    Store itla t -> do
```

Comment:
No need to fix this now, but in general this is about the scope size where I start to not like one-letter identifiers.